src/Entities/Schema.php

```diff
@@ -177,9 +177,24 @@ public function __construct()
             'sessions/{sessionId}' => function (array $content) {
                 return new Session($content);
             },
-            'login' => function (array $content) {
+            'users/login' => function (array $content) {
```

Comment:
That seems also unused, when user login he go to `sessions/{sessionId}`, right?

Can we use standard `POST sessions` here?